backend/glp/models.py

```
@@ -0,0 +1,307 @@
+        return f'{self.discord_id}: {self.glp_amount} GLP {self.status}'
+
+
+class GLPAdjustment(BaseModel):
+    """Append-only superuser correction. Separate from runs on purpose:
+    adjustments never participate in run ordering, digests or checkpoints."""
+
+    ORIGIN_CHOICES = [
+        (ORIGIN_BUILDER, 'Builder'),
+        (ORIGIN_COMMUNITY, 'Community'),
+        (ORIGIN_GENERAL, 'General'),
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)